dynamiclinks/DynamicLinksSwiftUIExample/DynamicLinksExampleApp.swift

```diff
@@ -43,8 +47,17 @@ struct DynamicLinksExampleApp: App {
               "The URL \(url) does not have an FDL Custom Scheme; should be handled as Universal URL."
             )
             Task {
-              if let dynamicLink: DynamicLink = await handleLink(url) {
-                receivedLinkModel = ReceivedLinkModel(receivedURL: url, dynamicLink: dynamicLink)
+              do {
+                if let dynamicLink: DynamicLink = try await handleLink(url) {
+                  receivedLinkModel = ReceivedLinkModel(
+                    receivedURL: url,
+                    dynamicLink: dynamicLink
+                  )
+                }
+              } catch UniversalURLError.notHandledByFDL {
+                print("FIRDynamicLinks is not handling the link.")
```

Comment:
Don't use Objective-C "FIR" prefix here and below.

Reply:
This code no longer exists but noted for the future.